docs/manual/docs/cmd/spo/tenant/tenant-recyclebinitem-restore.md

```diff
@@ -0,0 +1,39 @@
+# spo tenant recyclebinitem restore
+
+Restores the specified deleted Site Collection from Site Collection Recycle Bin
+
+## Usage
+
+```sh
+spo tenant recyclebinitem restore [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-u, --url`|URL of the Site Collection to restore
+`--wait`|Wait for the Site Collection to be restored before completing the command
+`--query [query]`|JMESPath query string. See [http://jmespath.org/](http://jmespath.org/) for more information and examples
+`-o, --output [output]`|Output type. `json|text`. Default `text`
+`--pretty`|Prettifies `json` output
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+!!! important
+    you have to be at least a SharePoint Administrator (not just an owner) to use this command.
```

Comment:
Let's adjust this statement to the standard statement we use in other tenant-level commands